builtin/aws/ecs/platform.go

```diff
@@ -1339,10 +1339,12 @@ func (p *Platform) resourceTargetGroupCreate(
 			createTargetGroupInput.HealthCheckPath = aws.String(p.config.HealthCheck.Path)
 		}
 
+		createTargetGroupInput.HealthCheckTimeoutSeconds = aws.Int64(5)
 		if p.config.HealthCheck.Timeout != 0 {
 			createTargetGroupInput.HealthCheckTimeoutSeconds = aws.Int64(p.config.HealthCheck.Timeout)
 		}
 
+		createTargetGroupInput.HealthCheckIntervalSeconds = aws.Int64(30)
```

Comment:
Should we verify `HealthCheckTimeoutSeconds` is lower than `HealthCheckIntervalSeconds` or do we just let AWS error there?

Reply:
This part of the `ConfigSet` function already handles that for us! The reason why this PR was still needed though is because `ConfigSet` will only check what the user sets in the `waypoint.hcl` - it wasn't checking the values that the AWS Go SDK was setting by default when one wasn't supplied.

https://github.com/hashicorp/waypoint/blob/8acf8277bbaef839f7d6e020d6acbf50c84c3faf/builtin/aws/ecs/platform.go#L134-L139

Reply:
👍